src/_button.scss

```diff
@@ -0,0 +1,57 @@
+.button {
+  box-sizing: border-box;
+  padding: 12px 25px;
+  margin: 5px 5px 5px 0;
+  font-family: $font__family--main;
+  font-size: $font__size--headings-least;
+  color: $color--light;
+  cursor: pointer;
+
+  &--light {
+    border: 1px solid $color--dark;
+    color: $color--dark;
+    background-color: $color--light;
+  }
+  &--dark {
+    background-color: $color--dark;
+    color: $color--light;
+    border: 1px solid $color--dark;
+  }
+
+  &--text {
+    background-color: $color--own;
+    padding: 10px 0;
+    opacity: 0.8;
+    border: 1px solid $color--own;
+  }
+}
+
+@media (min-width: 481px) {
+.button {
+  padding: 15px 30px;
+  font-size: $font__size--headings-least*1.5;
```

Comment:
Please add spaces before and after ` * `.

Reply:
Added.